src/rt/test/func/lambdabehaviour/lambdabehaviour.cc

```diff
@@ -0,0 +1,29 @@
+// Copyright Microsoft and Project Verona Contributors.
+// SPDX-License-Identifier: MIT
```

Comment:
It would be good if the tests actually had state in the closures to exercise that aspect.

It would also be good if you captured a smart pointer, to check that the destructors are correctly called.